src/open_inwoner/js/components/confirmation/index.js

```diff
@@ -11,15 +13,21 @@ class Confirmation {
     if (!this.real_submit) {
```

Comment:
`real_submit` is a strange name. Is there any other kind of `submit` field? I know it's not your change, but perhaps consider changing this (unless it makes sense to you and I just misunderstand).

Reply:
Yeah that's not mine 🙂 but I understand why they chose this name: this name is often used when pop-up modals pop up after clicking something on the pages that just LOOKS like a Submit button, but actually isn't - and this modal warns the user about the data they are about to submit/delete - so not until they actually click the confirm button in the modal, does the "real" POST happen. So I think I'd like to keep this name.

Reply:
Ah, that makes sense!